### PR TITLE
tofi: update to 0.9.1

### PR DESCRIPTION
**Pull request by LucasTavaresA**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl **cross**

### CODE REVIEW

File: srcpkgs/tofi/template [hunk 1]
```diff
@@ -12,7 +12,13 @@ license="MIT"
 homepage="https://github.com/philj56/tofi"
 changelog="https://raw.githubusercontent.com/philj56/tofi/master/CHANGELOG.md"
 distfiles="https://github.com/philj56/tofi/archive/refs/tags/v${version}.tar.gz"
-checksum=0a700f8b0291ef08b3544cd589157393101d52bbd1e7e6aeee8484f9d520a338
+checksum=dd55347b509af786f133def64f3a86f1db258b5e196de80e53d62827bdcc37bc
+
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	do_check() {
```
Comment (classabbyamp) [inline srcpkgs/tofi/template:18]: don't do this, use `make_check=no`

File: srcpkgs/tofi/template [hunk 2]
```diff
@@ -12,7 +12,12 @@ license="MIT"
 homepage="https://github.com/philj56/tofi"
 changelog="https://raw.githubusercontent.com/philj56/tofi/master/CHANGELOG.md"
 distfiles="https://github.com/philj56/tofi/archive/refs/tags/v${version}.tar.gz"
-checksum=0a700f8b0291ef08b3544cd589157393101d52bbd1e7e6aeee8484f9d520a338
+checksum=dd55347b509af786f133def64f3a86f1db258b5e196de80e53d62827bdcc37bc
+
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	# some tests fail on i686.
```
Comment (classabbyamp) [inline srcpkgs/tofi/template:18]: can you be more specific?

Comment (LucasTavaresA) [reply srcpkgs/tofi/template:18]: the only pattern after checking logs and looking at the tests, is that it fails checking options should be invalid

Comment (LucasTavaresA) [reply srcpkgs/tofi/template:18]: not sure if that's correct info or even useful

**Comment by classabbyamp** [comment]
I would make an issue about those test failures upstream, they are probably not dealing with 32-bit platforms properly

File: srcpkgs/tofi/template [hunk 3]
```diff
@@ -12,7 +12,12 @@ license="MIT"
 homepage="https://github.com/philj56/tofi"
 changelog="https://raw.githubusercontent.com/philj56/tofi/master/CHANGELOG.md"
 distfiles="https://github.com/philj56/tofi/archive/refs/tags/v${version}.tar.gz"
-checksum=0a700f8b0291ef08b3544cd589157393101d52bbd1e7e6aeee8484f9d520a338
+checksum=dd55347b509af786f133def64f3a86f1db258b5e196de80e53d62827bdcc37bc
+
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	# broken tests on i686, fails checking for invalid options.
```
Comment (classabbyamp) [inline srcpkgs/tofi/template:18]: let's add https://github.com/philj56/tofi/commit/667075f0920da3c2b353fbce54b6430c195ef031 as a patch instead

**Pull request merged**